mvlearn/decomposition/multiviewica.py

```diff
@@ -0,0 +1,182 @@
+        an inverse transform method to recover original data from reduced data.
+
+    multiview_output : bool, optional (default True)
+        If True, the `.transform` method returns one dataset per view.
+        Otherwise, it returns one dataset, of shape (n_samples, n_components)
+
+    random_state : int, RandomState instance or None, default=None
+        Used to perform a random initialization. If int, random_state is
+        the seed used by the random number generator; If RandomState
+        instance, random_state is the random number generator; If
+        None, the random number generator is the RandomState instance
+        used by np.random.
+
+    tol : float, default=1e-3
+        A positive scalar giving the tolerance at which
+        the un-mixing matrices are considered to have converged.
+
+    verbose : bool, default=False
+        Print information
+
+    Attributes
+    ----------
+    preproc : ViewTransformer-like instance
+        The fitted instance used for preprocessing
+    W_list : np array of shape (n_groups, n_components, n_components)
+        The unmixing matrices to apply on preprocessed data
+
+    See also
+    --------
+    groupica
+    permica
+
+    Notes
+    -----
+    Given each view :math:`X_i` It optimizes:
+
+        .. math::
+            l(W) = \frac{1}{T} \sum_{t=1}^T [\sum_k log(cosh(Y_{avg,k,t}))
+            + \sum_i l_i(X_{i,.,t})]
+
+    where
+
+        .. math::
+            l _i(X_{i,.,t}) = - log(|W_i|) + 1/(2 \sigma) ||X_{i,.,t}W_i -
+            Y_{avg,.,t}||^2,
+
+    :math:`W_i` is the mixing matrix for view  :math:`i`,
+    :math:`Y_{avg} = \frac{1}{n} \sum_{i=1}^n X_i W_i`, and :math:`\sigma`
+    is the noise level.
+
+    References
+    ----------
+    .. [#1mvica] Hugo Richard, Luigi Gresele, Aapo Hyvärinen, Bertrand Thirion,
+        Alexandre Gramfort, Pierre Ablin. Modeling Shared Responses in
+        Neuroimaging Studies through MultiView ICA. arXiv 2020.
+
+    Examples
+    --------
+    >>> from mvlearn.datasets import load_UCImultifeature
+    >>> from mvlearn.decomposition import MultiviewICA
+    >>> Xs, _ = load_UCImultifeature()
+    >>> ica = MultiviewICA(n_components=3, max_iter=10)
+    >>> sources = ica.fit_transform(Xs)
+    >>> print(sources.shape)
+    (6, 2000, 3)
+    """
+
+    def __init__(
+        self,
+        n_components=None,
+        noise=1.0,
+        max_iter=1000,
+        init="permica",
+        multiview_output=True,
+        random_state=None,
+        tol=1e-3,
+        verbose=False,
+        n_jobs=30,
+        preproc="pca",
+    ):
+        super().__init__(
+            n_components=n_components,
+            preproc=preproc,
+            random_state=random_state,
+            verbose=verbose,
+            multiview_output=multiview_output,
+        )
+        self.noise = noise
+        self.max_iter = max_iter
+        self.init = init
+        self.tol = tol
+
+    def fit_(self, Xs, y=None):
```

Comment:
This is just a helper function right, that the user would never call? If so, I think you should remove it from the docs by editing the decomposition.rst (https://github.com/mvlearn/mvlearn/blob/master/docs/references/decomposition.rst) to exclude this, and any other helper functions, from any class where they shouldn't actually be called by the user